app/components/TopCommunities/index.js

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { FormattedMessage } from 'react-intl';
+
+import { white, blue } from 'style-constants';
+import * as routes from 'routes-config';
+import messages from 'common-messages';
+
+import { getFormattedNum2 } from 'utils/numbers';
+import FollowCommunityButton from 'containers/FollowCommunityButton';
+
+import A from 'components/A';
+import H4 from 'components/H4';
+import Span from 'components/Span';
+import Base from 'components/Base';
+import Button from 'components/Button';
+import Img from 'components/Img';
+
+const BaseStyled = Base.extend`
+  padding: 15px 20px;
+`;
+
+const BackSide = styled.div`
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: ${white};
+  display: none;
+  border-radius: 5px;
+
+  > div {
+    position: relative;
+    height: 100%;
+    padding: 15px 20px;
+  }
+`;
+
+const AStyled = A.extend`
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  width: 100%;
+  height: 100%;
+  overflow: hidden;
+  word-break: break-all;
+
+  &:hover ${BackSide} {
+    display: block;
+  }
+`;
+
+const communitiesRoute = routes.communities();
+
+const TopCommunities = ({ communities, profile, account, userId }) => {
+  if (
+    account !== userId ||
+    !communities ||
+    !profile ||
+    profile.followed_communities[0]
+  ) {
+    return null;
+  }
+
+  return (
+    <div>
+      <H4 isHeader>Top communities</H4>
+      <div className="row">
+        {communities.map(x => (
+          <div className="col-xl-3 mb-2">
+            <AStyled to={communitiesRoute} href={communitiesRoute}>
+              <BaseStyled className="flex-grow-1" position="top">
+                <Img size={1.75} src={x.avatar} alt="comm_img" />
+                <p>
+                  <Span fontSize="16" bold>
```

Comment:
Either move font sizes to constants or create variations of Span component (SpanLarge, SpanMedium, SpanSmall).